exporters/metric/cortex/cortex.go

```diff
@@ -62,3 +68,78 @@ func InstallNewPipeline(config Config, options ...push.Option) (*push.Controller
 	global.SetMeterProvider(pusher.Provider())
 	return pusher, nil
 }
+
+// AddHeaders adds required headers as well as all headers in Header map to a http request.
+func (e *Exporter) AddHeaders(req *http.Request) {
+	// Cortex expects Snappy-compressed protobuf messages. These two headers are hard-coded as they
+	// should be on every request.
+	req.Header.Add("Content-Encoding", "snappy")
```

Comment:
There is no Prometheus Remote Write Version: 0.1.0 as one of the expected headers?